subprojects/core/src/integTest/groovy/org/gradle/api/tasks/options/TaskOptionIntegrationTest.groovy

```diff
@@ -321,4 +321,121 @@ Options
         then:
         result.assertTaskExecuted(':myTask').assertOutputContains('Serial: 4321')
     }
+
```

Comment:
What does this do? Do I get both `--bar` and `--foo`?
```
interface Intf1 {
    @Option(option="foo")
    Property<String> getFoo()
}
interface Intf2 {
    @Option(option="bar")
    Property<String> getFoo()
}
abstract class MyTask extends DefaultTask implements Intf1, Intf2 {
}
```

Reply:
This one is already tested, and yes both exist, but `--bar` has priority.